cobalt/dom/lottie_player.cc

```diff
@@ -298,9 +299,28 @@ void LottiePlayer::UpdateAnimationData() {
       return;
     }
 
+    auto isBase64Encoded = [](const std::string& url) -> bool {
+      size_t pos = url.find(',');
+      if (pos != std::string::npos) {
+        std::string beforeComma = url.substr(0, pos);
+        return beforeComma.find("base64") != std::string::npos;
+      }
+      return false;
+    };
+
+    // Force base64 encoding of plain/raw inputs, to ensure no URL escaping
```

Comment:
Should there be a TODO with a bug to remove the non-standard urls we use in the future?